### PR TITLE
Tweaks Explosive Holoparasites

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Explosive Holoparasites now have a 33% chance to explosively teleport attacked targets, doing damage to everyone near the target's appearance point.
tweak: Explosive Holoparasite bombs can now detonate when living mobs bump them.
/:cl:

### CODE REVIEW

**Comment by KorPhaeron** [comment]
LORE VIOLATION

**Comment by ChangelingRain** [comment]
Counterpoint; telescience bombs.

**Comment by KorPhaeron** [comment]
Counterpoint; Go read part IV

**Comment by drynwyn** [comment]
these aren't stands though, they are a Distinct Intellectual Property (tm)

**Comment by Zilenan91** [comment]
Does the explosive attack cause explosions at both the endpoint and where they got hit, because it sounds really absurdly powerful if it can't backfire.

**Comment by ChangelingRain** [comment]
Just at the endpoint.

**Comment by KingDairy** [comment]
sheer heart attack would be a good buff

**Comment by KorPhaeron** [comment]
>sheer heart attack would be a good buff

**Comment by Zilenan91** [comment]
@ChangelingRain Well yeah but if it's an explosion the holo basically has a 1/3 chance of KO'ing people every hit, I'd have thought that it would have the possibility of KOing the owner as well.

**Comment by ChangelingRain** [comment]
It's not an actual explosion it's just AoE brute damage

**Comment by Zilenan91** [comment]
Oh ok nevermind then

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
 /mob/living/simple_animal/hostile/guardian/Life() //Dies if the summoner dies
 	..()
-
+	update_health_hud() //we need to update our health display to match our summoner and we can't practically give the summoner a hook to do it
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: todo: render this comment wrong(sometime in the future)

**Comment by TechnoAlchemisto** [comment]
sometimes you have to make your own EU

**Pull request merged**